Cargo.toml

```
@@ -368,6 +369,10 @@ debug = true
 lto = true
 opt-level = "s"
 
+[[example]]
+name= "display_touch"
+required-features = ["fsmc_lcd"] #stm32f412, stm32f413
```

```suggestion
required-features = ["fsmc_lcd", "stm32f412"] # or stm32f413
```
then lets compile only with 412